### PR TITLE
[AWS] Add ELB Connection logs dashboard

### PR DESCRIPTION
**Pull request by agithomas**
- Enhancement

## Proposed commit message

AWS ELB dataset supports connection logs of the Application load balancers. Create a dashboard for visualising key metrics and information of connection logs

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [x] Testing of the dashboard with version 8.16.0 and 8.16.1
- [x] Testing of the dashboard content with data from pipeline tests as well as from real ALB data. 

## How to test this PR locally

- elastic-package build
- elastic-package stack up -v -d --services package-registry
- Perform package upgrade

## Related issues

- Relates https://github.com/elastic/integrations/issues/11770

## Screenshots

Screenshot 2024-12-11 at 12 06 16 PM

------

aws-load-balancer-overview

### CODE REVIEW

File: packages/aws/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.36.0"
+  changes:
+    - description: Add ELB connection logs dashboards for application load balancers.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```
Comment (harnish-elastic) [inline packages/aws/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/12030
```

File: packages/aws/img/filebeat-aws-alb-connection-overview.png
Comment (harnish-elastic) [inline packages/aws/img/filebeat-aws-alb-connection-overview.png:1]: Can we show `failed` connection request status as red color in `Connection Request Status` chart?

File: packages/aws/changelog.yml (hunk 1 above)
Comment (agithomas) [reply packages/aws/changelog.yml]: Addressed. Thanks

File: packages/aws/img/filebeat-aws-alb-connection-overview.png
Comment (harnish-elastic) [inline packages/aws/img/filebeat-aws-alb-connection-overview.png:1]: Minor suggestion:

Can we update the title from `Certificate Expired and Expiring in 30 days` to `Certificate Expired or Expiring in 30 days`?

**Review by harnish-elastic — CHANGES_REQUESTED** [review]
Minor nits given!

File: packages/aws/img/filebeat-aws-alb-connection-overview.png
Comment (agithomas) [reply packages/aws/img/filebeat-aws-alb-connection-overview.png:1]: Updated as suggested.

Comment (agithomas) [reply packages/aws/img/filebeat-aws-alb-connection-overview.png:1]: Updated as suggested.

**Review by harnish-elastic — APPROVED** [review]

**Comment by muthu-mps** [comment]
@agithomas - Why the latency is shown in metric chart? Is it possible to visualise the latency in line chart?

**Comment by agithomas** [comment]
> @agithomas - Why the latency is shown in metric chart? Is it possible to visualise the latency in line chart?

Updated as suggested.

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**